README.md

```diff
@@ -100,7 +100,12 @@ A glossary entry is structured like this:
         and the value may contain local links to other terms in this glossary
         (i.e., links starting with `#`)
         and/or links to outside sources.
+### Building a local test site
+Once you have finished creating your glossary entry it may be useful to build a local test site to check the syntax and formatting is correct. To do this you will need a working installation of Jekyll on your local machine (instructions for which can be found [here](https://jekyllrb.com/docs/installation/).
```

Comment:
wording change suggestion:

```suggestion
Once you have finished creating your glossary entry, you could preview the site on your computer to check that the syntax and formatting are correct. To do this you will need a working installation of Jekyll on your local machine (instructions for which can be found [here](https://jekyllrb.com/docs/installation/). 
```

Reply:
I'm also wondering if it'd be worth pointing to [The Carpentries lesson template setup instructions](https://carpentries.github.io/lesson-example/setup.html) as it is more comprehensive than Jekyll's. The issue might be that it might be distracting/confusing for people not familiar with The Carpentries to understand the relationship between glosario and our lessons.

Reply:
Maybe we could take just the portion of that page that describes the Jekyll stuff and have it be one page in the Glosario site. That way we don't confuse people with stuff about the lesson, but we do show them where they can find more comprehensive instructions—if they are so inclined.

Reply:
yes that sounds good, especially given that we are going to change the content of this page in the near future as we're going to move away from Jekyll for our lessons.

Reply:
I've been trying to get this working, but I haven't managed to get the CSS cooperating. Glosario doesn't use an external theme, so I tried pulling the relevant bits from the Carpentries theme, but something about the code blocks isn't right—and I can't tell what, even though I've inspected multiple pages.

Is there some part of those code block boxes that is actually created by knitr that would somehow still be missing if I recapitulated the html myself? See images below for what it should look like, vs what it does look like.

<img width="999" alt="Screen Shot 2021-04-07 at 21 33 11" src="https://user-images.githubusercontent.com/12277715/113930209-e5e30780-97e8-11eb-915d-81dbcd7d7dd1.png">

<img width="842" alt="Screen Shot 2021-04-07 at 21 29 24" src="https://user-images.githubusercontent.com/12277715/113930104-c946cf80-97e8-11eb-8074-d04c1e053a5d.png">

Reply:
I've created a new branch (add_setup) where I've added a markdown version of the Jekyll setup from this page. I have replaced the knitr stuff with generic GitHub markdown code rendering. We can discuss more styling in the draft PR: https://github.com/carpentries/glosario/pull/302.

Reply:
I think it's because we rely on bootstrap for the formatting and that it's missing here.